admin/led

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -eu
```

Comment:
It is really required bash?

Reply:
Yes, there is a bashims, such as [[.
All scripts use bash (except installer).

Reply:
I'm not sure that it is strongly required, but ok, let it be.
we can rewrite it later ;)